src/main/java/io/skai/okta/internshipstreamingserviceparserxml/scheduler/quartz.properties

```diff
@@ -0,0 +1,3 @@
+org.quartz.scheduler.instanceName = StreamingServiceScheduler
```

Comment:
don't create separate properties file, place this settings into regular application.properties file